src/Sulu/Bundle/ContentBundle/Twig/ContentTwigExtension.php

```diff
@@ -138,7 +138,7 @@ public function needsAddButtonFunction(PropertyInterface $property)
      */
     public function isMultipleTest($property)
     {
-        return $property->getMinOccurs() > 1;
+        return $property->getMinOccurs() > 1 || !is_null($property->getMaxOccurs());
```

Comment:
Wouldn't it be safer to ask if `maxOccurs` is bigger than `1` or `null`? The value of `minOccurs` doesn't really matter here, since it must be lower than `maxOccurs` (although I am not really sure if this is checked...).

Maybe @wachterjohannes can also add his opinion, since he wrote this part.